client/scripts.js

```diff
@@ -1,62 +1,69 @@
 // Use it to identify yourself when sending messages
-var serverToken = "";
+var serverToken = '';
 // Displayed name of current user
-var username = "test" + Date.now().valueOf().toString();
+var username = 'test' + Date.now().valueOf().toString();
 
 var chatWindow = document.getElementById('chatWindow');
 var messageWindow = document.getElementById('');
-var btnChatMessageSend = document.getElementById("btnMessageSend");
-var textAreaChatMessage = document.getElementById("textChatMessageField");
+var btnChatMessageSend = document.getElementById('btnMessageSend');
+var textAreaChatMessage = document.getElementById('textChatMessageField');
 
 // Create websocket connection
-const socket = new WebSocket("ws://localhost:8080", "chat");
+const socket = new WebSocket('ws://localhost:8080', 'chat');
 
 // Connection opened
-socket.addEventListener("open", function(event) {
-    console.log("Opened socket to the chat server, connecting to chat...");
-    connectToServer(socket);
+socket.addEventListener('open', function(event) {
+  console.log('Opened socket to the chat server, connecting to chat...');
+  connectToServer(socket);
 });
 
 // SEND button click function that will send message
-btnChatMessageSend.addEventListener("click", function(event) {
-    console.log("SEND btn clicked, server token:", serverToken);
-    // If there is no server token return
-    if(serverToken === "") return;
+btnChatMessageSend.addEventListener('click', function(event) {
+  console.log('SEND btn clicked, server token:', serverToken);
+  // If there is no server token return
+  if (serverToken === '') return;
 
-    var messageBody = textAreaChatMessage.value;
-    console.log("message body:", messageBody);
-    sendMessage(socket, serverToken, messageBody);
+  var messageBody = textAreaChatMessage.value;
+  console.log('message body:', messageBody);
+  sendMessage(socket, serverToken, messageBody);
 
-    // Clear text area
-    textAreaChatMessage.value = "";
+  // Clear text area
+  textAreaChatMessage.value = '';
 });
 
 function toggleMenu() {
-    var menuDisplay = document.getElementById('menuWindow');
-    var rotateImage = document.getElementById('btnMenu');
-    if (menuDisplay.style.display === 'block') {
-        rotateImage.style.transform = 'rotate(45deg)';
-        menuDisplay.style.display = 'none';
-    }
-    else {
-        rotateImage.style.transform = 'rotate(-90deg)';
-        menuDisplay.style.display = 'block';
-    }
+  var menuDisplay = document.getElementById('menuWindow');
+  var rotateImage = document.getElementById('btnMenu');
+  if (menuDisplay.style.display === 'block') {
+    rotateImage.style.transform = 'rotate(45deg)';
+    menuDisplay.style.display = 'none';
+  } else {
+    rotateImage.style.transform = 'rotate(-90deg)';
+    menuDisplay.style.display = 'block';
+  }
 }
 
+// User registration popup
+
+window.onload = function() {
+  document.getElementById('register-button').onclick = function() {
```

Comment:
When register-button is pressed:
- set var username to what is in text area in registration popup
- call connectToServer(socket)